helix-core/src/main/java/org/apache/helix/monitoring/metrics/model/LatencyMetric.java

```diff
@@ -28,8 +28,6 @@
  * how long a particular stage in the logic took in milliseconds.
  */
 public abstract class LatencyMetric extends HistogramDynamicMetric implements Metric<Long> {
-  protected long _startTime;
-  protected long _endTime;
```

Comment:
This should be in a separate PR with changes involving the other Metric classes.